src/mapped-types.spec.ts

```diff
@@ -95,6 +97,22 @@ it('Omit', () => {
   testType<Omit<Props | NewProps, 'age'>>();
 });
 
+// @dts-jest:group PickByValue
+it('PickByValue', () => {
+  // @dts-jest:pass:snap
+  testType<PickByValue<Props, 'age'>>();
```

Comment:
All the test are incorrect, you should use it normally as you did in the readme example, for instance:
```
testType<PickByValue<Props, string | number>>();

testType<PickByValue<Props | NewProps, string | number>>();
```

Please fix all the added tests

Reply:
I updated tests as you suggested, but do my tests really checks anything.

I didn't write expected result anywhere. Since, I didn't write type tests nor jest snapshot before, I can't be sure.